resources/package.rb

```diff
@@ -45,6 +46,7 @@
 
   remote_file basename do
     source r.name
+    checksum r.checksum
```

Comment:
Can you add a conditional here? Otherwise it will break for folks that don't provide a checksum.

Reply:
Realizing this doesn't need this, and instead we should not add a test for checksum to the test recipe.